src/util/i18nFormatting.js

```diff
@@ -0,0 +1,24 @@
+const formatFlags = {
+  EN_HANDLE_AN: 'en-handle-an',
+  CAPITALIZE: 'capitalize'
+}
+
+function getVariationOfAOrAn(value) {
+  const vowelishLetters = ['a', 'e', 'i', 'o', 'u', 'h'];
+  const isVowelish = vowelishLetters.find(l => value.substring(0, 1) === l);
+  return isVowelish ? 'an' : 'a';
+}
+
+// eslint-disable-next-line no-unused-vars
+export default function(value, format, lng) {
```

Comment:
Also, I’m a fan of still giving default exports a name (so `export default function i18nFormatting(…) {` although maybe I would change the module/function name to something like verb-y like `applyCustomI18nFormatters`)